game.js

```diff
@@ -122,22 +122,34 @@ class Game {
                 && clock.pause && clock.pause.paused && clock.pause.pause_control
                 && !clock.pause.pause_control["stone-removal"] && !clock.pause.pause_control.system && !clock.pause.pause_control.weekend
                 && !clock.pause.pause_control["vacation-" + clock.black_player_id] && !clock.pause.pause_control["vacation-" + clock.white_player_id]) {
-                if (config.DEBUG) this.log("Pausing not allowed. Resuming game.");
-                this.resumeGame();
+                if (config.DEBUG) {
+                    const noPauseMg = "Pausing not allowed. Resuming game.";
+                    this.log(noPauseMg);
+                    this.sendChat(noPauseMg)
```

Comment:
Why does ` if (config.DEBUG)` affect `this.sendChat(noPauseMg)`? Also `;` missing here.

Reply:
indeed, i actually enhanced it by using a function now !!!